content/patterns/radio/examples/radio.html

```diff
@@ -25,119 +25,115 @@
+            <div role="radio" aria-checked="false" tabindex="-1">シンクラスト</div>
           </div>
 
           <div role="radiogroup" aria-labelledby="group_label_2" id="rg2">
-            <h3 id="group_label_2">Pizza Delivery</h3>
-            <div role="radio" aria-checked="false" tabindex="0">Pickup</div>
+            <h3 id="group_label_2">ピザのお受け渡し</h3>
+            <div role="radio" aria-checked="false" tabindex="0">お持ち帰り</div>
 
-            <div role="radio" aria-checked="false" tabindex="-1">Home Delivery</div>
+            <div role="radio" aria-checked="false" tabindex="-1">宅配</div>
 
-            <div role="radio" aria-checked="false" tabindex="-1">Dine in</div>
+            <div role="radio" aria-checked="false" tabindex="-1">イートイン</div>
           </div>
         </div>
-        <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
+        <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="終了"></div>
       </section>
 
       <section>
-        <h2>Accessibility Features</h2>
+        <h2>アクセシビリティ機能</h2>
         <ul>
-          <li>Uses CSS attribute selectors for synchronizing <code>aria-checked</code> state with the visual state indicator.</li>
+          <li><code>aria-checked</code> ステートと視覚的な状態表示を同期させるために、CSS の属性セレクタを使用します。</li>
           <li>
-            Uses CSS <code>:hover</code> and <code>:focus</code> pseudo-classes for styling visual keyboard focus and hover.
+            視覚的なキーボードフォーカス及びホバーのスタイリングに、CSS の <code>:hover</code> 及び <code>:focus</code> 疑似クラスを使用します。
             <ul>
-              <li>The focus indicator encompasses both the radio button and label, making it easier to perceive which option is being chosen.</li>
-              <li>Hover changes background of both the radio button and label, making it easier to perceive that clicking either the label or button will activate the radio button.</li>
-              <li>The cursor is changed to a pointer when hovering over the radio button to help people with visual impairments identify it as an interactive element.</li>
+              <li>フォーカスインジケータはラジオボタン及びラベルの両方を囲むため、どのオプションが選ばれているかを、認識しやすくなっています。</li>
+              <li>ホバーはラジオボタン及びラベルの両方の背景を変更するため、ラベルあるいはボタンのいずれをクリックしてもラジオボタンが作動することを、認識しやすくなっています。</li>
+              <li>ラジオボタンの上にホバーすると、カーソルの形状がポインタに変わるため、視覚障害のある人々がそれをインタラクティブな要素であると識別するのに役立ちます。</li>
             </ul>
           </li>
           <li>
-            Because transparent borders are visible on some systems with high contrast enabled, only the focused element has a visible border.
-            When focusable elements are not focused, they have a 0-width border and additional padding equal in width to the border that is used to indicate focus.
+            透明なボーダーは、ハイコントラストが有効になっているシステムの一部で視覚的に見えてしまうことがあるため、フォーカスされた要素にのみ視覚的なボーダーを持たせています。フォーカス可能な要素がフォーカスされていない場合は、その要素に対して幅が 0 のボーダー及び、フォーカスを示すのに使われるボーダーと同じ幅の追加的なパディングを持たせています。
```

Comment:
好み問題かもしれませんが、「システムの一部」と書くとシステムの中の一部であると捉えられてしまうかもしれません。「一部のシステム」とすれば良いかも、ということなのですが、ここでのsomeのニュアンスはすでに「ことがある」という表現で伝えられているような気もします。

「visible border」という表現は、視覚的なというよりは目に見えるということを伝えるための表現かなと思うので、「可視のボーダー」などとして見るのはどうでしょうか。ちょっと慣れない表現ですかね・・

```suggestion
            透明なボーダーは、ハイコントラストが有効になっているシステムで表示されてしまうことがあるため、フォーカスされた要素にのみ可視のボーダーを持たせています。フォーカス可能な要素がフォーカスされていない場合は、その要素に対して幅が 0 のボーダー及び、フォーカスを示すのに使われるボーダーと同じ幅の追加的なパディングを持たせています。
```